### PR TITLE
[GITHUB-3] version_compare

### PR DESCRIPTION
**Pull request by msahota14**
#3

### CODE REVIEW

File: molecule/default/create.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   hosts: localhost
   connection: local
   gather_facts: false
-  no_log: "{{ not lookup('env', 'MOLECULE_DEBUG') | bool }}"
+  # no_log: "{{ not lookup('env', 'MOLECULE_DEBUG') | bool }}"
```
Comment (lobsterdore) [inline molecule/default/create.yml]: Can you uncomment this? And the one below as well.

**Review by lobsterdore — APPROVED** [review]
LGTM 👍

File: molecule/default/create.yml (hunk 1 above)
Comment (msahota14) [reply molecule/default/create.yml]: updated

**Comment by lobsterdore** [comment]
Just waiting for the tests to pass.

**Pull request merged**